### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
Added landing Page and Navbar Component.

### CODE REVIEW

**Review by ivinayakg — COMMENTED** [review]
The code works fine and looks good but there are couple of changes I suggest. 
1. Import the css link of your component library (it's required by neog) the only css you should write is to extend the styling or change
2. Break the html into components for example on the homepage you are writing all the html into one component that will reduce the flexibility of every component and you will end up using them repeatedly like this only. 
3. Your components (which you will make after this) should take the data from props.

File: src/pages/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
+import "../styles/homepage.css";
+
+const HomePage = () => {
+    return(
+        <div>
+            <main>
+        <section className="section section-main">
+            <div>
+            <img className="image-section"
+                src="https://cdn.shortpixel.ai/spai/q_+ret_img+to_webp/https://www.banarasisaree.com/wp-content/uploads/2021/11/slider-home-wedding-2nw.jpg"
```
Comment (chanchal-panpaliya) [inline src/pages/HomePage.js:10]: you can also use below code for image :--
import img_name from "..../...//path" 
.
.

File: src/components/navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+const Navbar = () => {
+    const [isOpen, setIsOpen] = useState(false);
+    const openNav = () => {
+        setIsOpen(!isOpen)
+    }
+return(
+    <div>
+        <nav className="navbar-ecom">
+            <a href="../index.html" className="brand-title-ecom">fastcart</a>
```
Comment (chanchal-panpaliya) [inline src/components/navbar/Navbar.jsx:13]: use link tag - react-router-dom for navigation

Comment (Mohasin-Haque) [reply src/components/navbar/Navbar.jsx:13]: I will add routers once i finished up with the UI part.

**Review by ivinayakg — COMMENTED** [review]
Good structuring and components building, go ahead and merge it

**Pull request merged**